### PR TITLE
21- Card Sorting Bug Fix

### PR DESCRIPTION
**Pull request by madeleine-patience**
https://github.com/madeleine-patience/Considerate-Cat-Version-2-With-Next/issues/21

# Pull Request Title

**🐾Description of Purrrrr Request🐾**
I have updated the way the data is called from supabase by ascending order and ID.  There was a small bug in one of the functions calling the cards as well. 

### Type of Change

- [ ] Mew Feature added
- [🐈 ] Bug fixed
- [🐈 ] Re-fur-ctor / Cleanup

### How did you purrrsonally test this
Locally

### Checklist

- [ ] Tests added or updated
- [ ] Documentation updated
- [🐈 ] Follows coding standards
- [🐈 ] Ready fur review

### CODE REVIEW

File: hooks/fetchTarotDeck.ts [hunk 1]
```diff
@@ -27,8 +27,9 @@ function useFetchTarotDeck(): TarotDeckProps {
         setLoading(true);
         const { data, error } = await supabase
           .from('tarotCardInformation')
-          .select('*');
-
+          .select('*')
```
Comment (arthvadrr) [inline hooks/fetchTarotDeck.ts:30]: **question:** Did we need the * for this call?

File: pages/CardDirectoryPage.tsx [hunk 2]
```diff
@@ -1,18 +1,18 @@
 import { Box, Typography, useMediaQuery, useTheme } from '@mui/material';
+import type { Theme } from '@mui/system';
+import type { ReactElement } from 'react';
```
Comment (arthvadrr) [inline pages/CardDirectoryPage.tsx:3]: **thought | nitpick:** We should establish a standard order for imports/types

File: pages/CardDirectoryPage.tsx [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 import { DisplayTarotCards } from '../components/displayTarotCards';
-import { TarotDeckContext } from '../context/TarotDeckContext';
-import { LoadingPage } from '../components/loadingPage/LoadingPage';
-import { useCardDirectory } from '../hooks/useCardDirectory';
-import { useTarotCard } from '../hooks/useTarotCard';
-import { Header } from '../components/header';
 import ElmerCircleIcon from '../components/elmerCircleIcon/ElmerCircleIcon';
 import FlowerFooter from '../components/flowerFooter/FlowerFooter';
+import { Header } from '../components/header';
```
Comment (arthvadrr) [inline pages/CardDirectoryPage.tsx:7]: Same here

File: pages/CardDirectoryPage.tsx [hunk 4]
```diff
@@ -44,9 +44,14 @@ export default function FirstPost(): ReactElement {
     displayCardBySuit(cards);
   }
 
+  function findCardById(deck: TarotCard[], id: number): TarotCard | undefined {
```
Comment (arthvadrr) [inline pages/CardDirectoryPage.tsx:47]: **praise:** Magnifique!

**Review by arthvadrr — APPROVED** [review]

File: hooks/fetchTarotDeck.ts (hunk 1 above)
Comment (madeleine-patience) [reply hooks/fetchTarotDeck.ts:30]: Seems to be fine without, removed!

File: pages/CardDirectoryPage.tsx (hunk 2 above)
Comment (madeleine-patience) [reply pages/CardDirectoryPage.tsx:3]: Its kinda weird it seems like the linter is suddenly kicking in and organizing them all. Not sure why. Ill open a separate PR for this and see whats happeneing.

File: pages/CardDirectoryPage.tsx (hunk 3 above)
Comment (madeleine-patience) [reply pages/CardDirectoryPage.tsx:7]: Will resolve in separate pr

**Pull request merged**